<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="input/objects" default="/perception/object_recognition/detection/clustering/objects_with_feature"/>
  <arg name="output/objects" default="/perception/object_recognition/detection/clustering/validated_objects_with_feature"/>

  <arg name="pointcloud_intensity_based_validator_param_path" default="$(find-pkg-share detected_object_validation)/config/pointcloud_intensity_based_validator.param.yaml"/>

  <node pkg="detected_object_validation" exec="pointcloud_intensity_based_validator_node" name="pointcloud_intensity_based_validator" output="screen">
    <remap from="input/objects" to="$(var input/objects)"/>
    <remap from="output/objects" to="$(var output/objects)"/>
    <param name="input_obstacle_pointcloud" value="false"/>
    <param from="$(var pointcloud_intensity_based_validator_param_path)"/>
  </node>

</launch>
